project_name/settings/base.py

```diff
@@ -127,6 +127,6 @@
 # Labs Accounts Settings
 
 PLATFORM_ACCOUNTS = {
-    'REDIRECT_URI': 'https://example.pennlabs.org/accounts/callback/',
+    'REDIRECT_URI': 'https://{}/accounts/callback/'.format(BACKEND_DOMAIN),
```

Comment:
Let's use f-strings here

Reply:
I also just realized, we have a weird mix of where we're defining settings. For example the db is defined in `base.py` with a fallback to dev settings, but then we have things like the backend/frontend_domain variables which are in `base.py` but default to dummy entries. We should probably try to standardize this.